### PR TITLE
Add `.sh` to javasrc2cpg start script.

### PR DESCRIPTION
**Pull request by ml86**
All other start scripts have also the `.sh` extension. No reason for an
exception here.

### CODE REVIEW

File: console/src/main/scala/io/joern/console/cpgcreation/JavaSrcCpgGenerator.scala [hunk 1]
```diff
@@ -11,7 +11,7 @@ import scala.util.Try
 /** Source-based front-end for Java
   */
 case class JavaSrcCpgGenerator(config: FrontendConfig, rootPath: Path) extends CpgGenerator {
-  private lazy val command: Path = if (isWin) rootPath.resolve("javasrc2cpg.bat") else rootPath.resolve("javasrc2cpg")
+  private lazy val command: Path = if (isWin) rootPath.resolve("javasrc2cpg.bat") else rootPath.resolve("javasrc2cpgi.sh")
```
Comment (max-leuthaeuser) [inline console/src/main/scala/io/joern/console/cpgcreation/JavaSrcCpgGenerator.scala:14]: Typo (-i)

**Review by max-leuthaeuser — APPROVED** [review]

**Comment by mpollmeier** [comment]
you're probably aware anyway, but just in case, some of the downstream places that need to be adapted:
https://github.com/ShiftLeftSecurity/codescience/blob/master/javasrc2cpg/build.sbt#L14
https://github.com/ShiftLeftSecurity/codescience/blob/master/semanticcpg-ext/src/test/scala/io/shiftleft/codepropertygraph/LanguageFrontend.scala#L277
https://github.com/ShiftLeftSecurity/codescience/blob/master/semanticcpg-ext/src/test/scala/io/shiftleft/codepropertygraph/LanguageFrontend.scala#LL277C4-L277C4

**Comment by fabsx00** [comment]
What's up with this one?

**Comment by ml86** [comment]
Closed since we dont want to adjust all the downstream users at the moment.

**Pull request closed without merging**